docs/atomic-quick-start-tutorial.adoc

```diff
@@ -0,0 +1,224 @@
+# Install our atomic enterprise repo
+$ sudo wget -O /etc/yum.repos.d/atomic-enterprise.repo http://mirror.ops.rhcloud.com/atomic/mirror/.atomic-enterprise-early-1/atomic-enterprise.repo
+
+# Install EPEL (For ansible)
+$ sudo yum -y install http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
+$ sudo sed -i -e "s/^enabled=1/enabled=0/" /etc/yum.repos.d/epel.repo
+-------------------------------------------------------------------------------------------------------------------------------
+
+8 Install ansible Package
+-------------------------
+[source,bash]
+-----------------------------------------------
+$ sudo yum -y --enablerepo=epel install ansible
+-----------------------------------------------
+
+9 Setup hosts
+-------------
+[source,bash]
+----------------------------------------------------------------------------------
+$ sudo hostnamectl set-hostname ae-master.example.com
+$ sudo bash -c “echo ‘${YOUR IP ADDRESS}    ae-master.example.com’ >> /etc/hosts”
+----------------------------------------------------------------------------------
+
+10 Clone Repos
+--------------
+[source,bash]
+----------------------------------------------------------------------------------
+cd /root
+git clone https://github.com/projectatomic/atomic-enterprise-training.git training
+git clone https://github.com/projectatomic/atomic-enterprise-ansible.git
+----------------------------------------------------------------------------------
+
+11 Setup SSH environment (for ansible)
+--------------------------------------
+[source,bash]
+----------------------------------
+ssh-keygen
+# Accept the defaults
+ssh-copy-id ae-master.example.com
+----------------------------------
+
+Sample output:
+
+[source,bash]
+---------------------------------------------------------------------------------------------------------------
+[root@ae-master ~]# ssh-copy-id ae-master.example.com
+The authenticity of host 'ae-master.example.com (192.168.122.239)' can't be established.
+ECDSA key fingerprint is 67:ca:e2:c2:eb:e0:fd:6c:de:74:bb:aa:6c:25:07:a7.
+Are you sure you want to continue connecting (yes/no)? yes
+/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed
+/bin/ssh-copy-id: INFO: 1 key(s) remain to be installed -- if you are prompted now it is to install the new keys
+root@ae-master.example.com's password:
+
+Number of key(s) added: 1
+---------------------------------------------------------------------------------------------------------------
+
+Now try logging into the machine, with:   "ssh 'ae-master.example.com'"
+and check to make sure that only the key(s) you wanted were added.
+
+12 setup ansible installer
+--------------------------
+[source,bash]
+------------------------------------------------------------------------------------------------------
+/bin/cp -r ~/training/eap-latest/ansible/* /etc/ansible/
```

Comment:
The ansible things are removed from the training repo since this commit:
https://github.com/projectatomic/atomic-enterprise-training/commit/11ce8cb270579de0ca0fb9878f77d00e4d65a2c2